### PR TITLE
Fix a typo in logging file

### PR DESCRIPTION
**Pull request by l-Legacy-l**
This removes a bad character which is causing logging not working using the content as logging configuration

### CODE REVIEW

File: geowebcache/web/src/main/resources/log4j2-rollingfile.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <CustomLevels>
       <CustomLevel name="CONFIG" intLevel="450" />
       <CustomLevel name="FINEST" intLevel="700" />
-    </CustomLevels>>
```
Comment (aaime) [inline geowebcache/web/src/main/resources/log4j2-rollingfile.xml:6]: Yes, thanks!

File: geowebcache/web/src/main/resources/log4j2-rollingfile.xml [hunk 2]
```diff
@@ -43,4 +43,4 @@
             <AppenderRef ref="Logfile"/>
         </Root>
     </Loggers>
-</Configuration>
\ No newline at end of file
+</Configuration>
```
Comment (aaime) [inline geowebcache/web/src/main/resources/log4j2-rollingfile.xml:46]: Please keep the newline at the end of file

Comment (l-Legacy-l) [reply geowebcache/web/src/main/resources/log4j2-rollingfile.xml:46]: Done 👍

**Comment by l-Legacy-l** [comment]
any news ?

**Comment by aaime** [comment]
Lost track of it. Merged!

**Pull request merged**